loops.js

```diff
@@ -51,3 +68,17 @@
 // >> 9
 // ==========================================
 
+for (let i = 0; i < 10; i++){
+    if (i <= 2){
+        console.log(i)
+    }
+
+
+    if (i >= 3 && i < 6){
+        console.log('>'+(i))
+    }
+
+    if (i >= 6){
+        console.log('>>'+(i))
+    }
+}
```

Comment:
for( let i=0; i<10; i++){
    if(i<3){
        console.log(i)
    }
    else if(i<7){
        console.log(">"+i)
    }
     else{
        console.log(">>"+i)
    }
}

------
dit is wat ik had